Resources/Prototypes/Entities/Clothing/OuterClothing/coats.yml

```
@@ -31,6 +31,28 @@
         Piercing: 0.70
         Heat: 0.80
 
+- type: entity
+  parent: ClothingOuterStorageBase
+  id: ClothingOuterCoatDetectiveLoadout
+  name: detective trenchcoat
+  description: A rugged canvas trenchcoat, designed and created by TX Fabrication Corp. Wearing it makes you feel for the plight of the Tibetans.
+  components:
+  - type: Sprite
+    sprite: Clothing/OuterClothing/Coats/detective.rsi
+  - type: Clothing
+    sprite: Clothing/OuterClothing/Coats/detective.rsi
+  - type: StorageFill
+    contents:
+    - id: SmokingPipeFilledTobacco
+    - id: FlippoLighter #Not the steal objective, only difference from normal detective trenchcoat
+  - type: Armor #same as regular sec armor
+    modifiers:
+      coefficients:
+        Blunt: 0.70
+        Slash: 0.70
+        Piercing: 0.70
+        Heat: 0.80
```

Just use the regular detective trenchcoat as the parent and change the storagefill component to have the desired contents.
```suggestion
- type: entity
  parent: ClothingOuterCoatDetective
  id: ClothingOuterCoatDetectiveLoadout
  components:
  - type: StorageFill
    contents:
    - id: SmokingPipeFilledTobacco
    - id: FlippoLighter #Not the steal objective, only difference from normal detective trenchcoat
```